src/pages/BtecGuide/index.tsx

```diff
@@ -0,0 +1,586 @@
+// Import libraries
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { HashLink as Anchor } from 'react-router-hash-link';
+import styled from 'styled-components';
+// Components
+import { Alert } from '../../components/Alert';
+import { Heading } from '../../components/Heading';
+import { Link } from '../../components/Link';
+import { PageTemplate } from '../../components/PageTemplate';
+import { Text } from '../../components/Text';
+import { Code } from '../../components/Code';
+
+const ComponentStyles = styled.div`
+  * {
+    scroll-behavior: smooth;
+    scroll-margin-top: 2rem;
+  }
+  a {
+    text-decoration: none;
+  }
+  p,
+  ol,
+  ul {
+    color: ${(p: any) => p.theme.blue.dark};
+  }
+  li {
+    font-weight: 300;
+  }
+  strong {
+    font-weight: 500;
+  }
+`;
+
+const CodeBlock = styled.code`
+  display: block;
+  padding: 20px;
+  font-family: Courier, sans-serif;
+  font-size: 1em;
+  line-height: 1.3;
+  color: ${(p: any) => p.theme.blue.dark};
+  background-color: #d0ddee;
+  border-radius: 6px;
+  margin-block: 20px;
+  overflow: scroll;
```

Comment:
```suggestion
  overflow: auto;
```

to have scrollbars only when necessary
![image](https://user-images.githubusercontent.com/468158/220176733-00f14791-d524-499a-a6b7-a926c2851771.png)